src/Microsoft.AspNetCore.SignalR/SignalRAppBuilderExtensions.cs

```diff
@@ -32,9 +32,14 @@ public HubRouteBuilder(SocketRouteBuilder routes)
             _routes = routes;
         }
 
-        public void MapHub<THub>(string path) where THub : Hub
+        public void MapHub<THub>(string path) where THub : Hub<IClientProxy>
```

Comment:
I think it was related to the other change. Seems no longer needed when I dropped the TClient.